Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN mkdir /tmp/azurecli \
 
 RUN curl -fsSL https://get.docker.com/ | sh
 
-ENV KUBECTL_VERSION 1.6.6
+ENV KUBECTL_VERSION 1.7.5
```

Comment:
@seanknox @lachie83 are we confident that this is the best kubectl version for testing all supported versions of k8s? (i.e., are we expressing with this change that it's preferable to have newer kubectl's talk to older k8s clusters vs older kubectl's talk to newer k8s clusters?

Reply:
Dockerfile used only in dev workflow AFAIK, and yes I think it's better for dev workflow to orient toward newer versions. Not attached to this, can remove in this PR.

Reply:
👍 I actually like it here.